karma.conf.js

```diff
@@ -0,0 +1,65 @@
+// Karma configuration
+
+module.exports = function (config) {
+    config.set({
+
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: '',
```

Comment:
How about using Proxies and / or Path mappings additionally? That could help us keeping the paths short and easy.

Take a look at:

https://karma-runner.github.io/0.13/config/configuration-file.html (Proxies and paths)
https://karma-runner.github.io/0.13/config/files.html (Loading Assets)